<?xml version="1.0" encoding="UTF-8"?>
<!--
     Hlavne okno aplikacie.
     Autori:
       Lisik Miroslav <xlisik00@stud.fit.vutbr.cz>
       Pokorny Fridolin <fridex.devel@gmail.com>
-->
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PN2012</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_Save_as"/>
    <addaction name="action_CloseTab"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Preferences">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="action_Pref"/>
   </widget>
   <widget class="QMenu" name="menu_Repository">
    <property name="title">
     <string>&amp;Repository</string>
    </property>
    <addaction name="action_Connect"/>
    <addaction name="action_Browse"/>
    <addaction name="action_Remote_Save"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_Help"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Preferences"/>
   <addaction name="menu_Repository"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="editToolBar">
   <property name="windowTitle">
    <string>Edit</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPlaces"/>
   <addaction name="actionTransitions"/>
   <addaction name="actionArrows"/>
   <addaction name="actionMove"/>
   <addaction name="actionEditObject"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionStep"/>
   <addaction name="actionRun"/>
   <addaction name="actionSimlog"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Connect">
   <property name="text">
    <string>&amp;Connect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Browse">
   <property name="text">
    <string>&amp;Browse</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_Remote_Save">
   <property name="text">
    <string>&amp;Remote Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Help">
   <property name="text">
    <string>&amp;Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_Save_as">
   <property name="text">
    <string>Save &amp;As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionPlaces">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/files/place.png</normaloff>:/files/place.png</iconset>
   </property>
   <property name="text">
    <string>Places</string>
   </property>
   <property name="toolTip">
    <string>Draw Places</string>
   </property>
  </action>
  <action name="actionTransitions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/files/transition.png</normaloff>:/files/transition.png</iconset>
   </property>
   <property name="text">
    <string>Transitions</string>
   </property>
   <property name="toolTip">
    <string>Draw Transitions</string>
   </property>
  </action>
  <action name="actionArrows">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/files/arrow.png</normaloff>:/files/arrow.png</iconset>
   </property>
   <property name="text">
    <string>Arrows</string>
   </property>
   <property name="toolTip">
    <string>Draw Arrows</string>
   </property>
  </action>
  <action name="actionMove">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/files/move.png</normaloff>:/files/move.png</iconset>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
   <property name="toolTip">
    <string>Move Drawing</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/files/delete.png</normaloff>:/files/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete Drawing</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/files/run.png</normaloff>:/files/run.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="toolTip">
    <string>Run Simulation</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/files/step.png</normaloff>:/files/step.png</iconset>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
   <property name="toolTip">
    <string>Step Simulation</string>
   </property>
  </action>
  <action name="actionSimlog">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/files/simlog.png</normaloff>:/files/simlog.png</iconset>
   </property>
   <property name="text">
    <string>Simlog</string>
   </property>
   <property name="toolTip">
    <string>View Simlog</string>
   </property>
  </action>
  <action name="action_Pref">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionEditObject">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/files/edit.png</normaloff>:/files/edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="toolTip">
    <string>Edit</string>
   </property>
  </action>
  <action name="action_Move">
   <property name="text">
    <string>&amp;Move</string>
   </property>
  </action>
  <action name="action_Place">
   <property name="text">
    <string>&amp;Place</string>
   </property>
  </action>
  <action name="action_Transition">
   <property name="text">
    <string>&amp;Transition</string>
   </property>
  </action>
  <action name="action_Arrow">
   <property name="text">
    <string>&amp;Arrow</string>
   </property>
  </action>
  <action name="action_Edit">
   <property name="text">
    <string>Edit &amp;Object</string>
   </property>
  </action>
  <action name="action_Delete">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
  </action>
  <action name="actionPetri_net_example_1">
   <property name="text">
    <string>Petri net example 1</string>
   </property>
  </action>
  <action name="actionPetri_net_example_2">
   <property name="text">
    <string>Petri net example 2</string>
   </property>
  </action>
  <action name="actionPetri_net_example_3">
   <property name="text">
    <string>Petri net example 3</string>
   </property>
  </action>
  <action name="actionPetri_net_example_4">
   <property name="text">
    <string>Petri net example 4</string>
   </property>
  </action>
  <action name="action_CloseTab">
   <property name="text">
    <string>CloseTab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
